src/js/media-container.js

```diff
@@ -366,7 +360,7 @@ class MediaContainer extends window.HTMLElement {
       // so, only run pointermove for mouse
       if (e.pointerType !== 'mouse') return;
 
-      if (!containsComposedNode(this, e.target)) return;
+      if (e.target === this) return;
```

Comment:
revert to this 
https://github.com/muxinc/media-chrome/commit/ac767acdc4218d42199e4abc6e1ebae2030c5f8a#diff-a093df03191bf45d0036001fefb7ca7217096f130642377d56e0cd90312b3e23R341

`containsComposedNode` did not work if `mux-video` was the target while it was not a direct child of media-controller but a "virtual" child via slots